dumux-precice/couplingadapter.cc

```diff
@@ -48,15 +48,14 @@ void CouplingAdapter::announceQuantity(const precice::string_view &meshName,
     dataMap_.insert(std::make_pair(key, dataValues));
 }
 
-int CouplingAdapter::getMeshDimensions(
-    const precice::string_view &meshName) const
+int CouplingAdapter::getMeshDimensions(const std::string &meshName) const
 {
     assert(wasCreated_);
     return precice_->getMeshDimensions(meshName);
 }
 
-void CouplingAdapter::setMesh(const precice::string_view &meshName,
-                              precice::span<const double> positions)
+void CouplingAdapter::setMesh(const std::string &meshName,
+                              std::vector<double> &positions)
```

Comment:
Do we want to stick to const positions? Could also make other data vectors and strings const.
```suggestion
                              const std::vector<double> &positions)
```